Cloudinary/Classes/Core/Features/UploadWidget/WidgetImages/BackIconInstractions.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  BackIconInstractions.swift
+//
+//  Copyright (c) 2020 Cloudinary (http://cloudinary.com)
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in all
+//  copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+//  SOFTWARE.
+//
+
+import UIKit
+
+class BackIconInstractions : CLDImageDrawingInstractions {
```

Comment:
Should be `instructions` (with a `u`).

Also - Isn't there a way to define an SVG like resource instead of using code to generate icons?

Reply:
@nitzanj about the name you're right and i'll fixit today

About the SVG, we're currently supporting SPM which is restricted at the moment to source code content only, in October it will be possible to change the implementation to work with resource files, including images, SVG, xib and such, however for now this is what we have, i didn't want to make the code DL an asset, as this is even a worst solution in the long run